pdns/backends/gsql/gsqlbackend.cc

```diff
@@ -1013,7 +1041,11 @@ bool GSQLBackend::get(DNSResourceRecord &r)
     } catch (SSqlException &e) {
       throw PDNSException("GSQLBackend get: "+e.txtReason());
     }
-    extractRecord(row, r);
+    try {
+      extractRecord(row, r);
+    } catch (...) {
+      goto skiprow;
```

Comment:
Ok, so the big question, strictly for alpha1, is "silently hide broken records" versus "kill backend on a broken record". I'll think about this.